### DISCUSSION TITLE
What's the use of Best Position column in the Team tab?

### DISCUSSION THREAD

**Discussion by tiagompb** [ho-dev/HattrickOrganizer#1810] [General]
Hi everyone,

I'm asking in this discussion what's the use of the best position column in the Team tab? I really don't get it! And why am I asking? Because, I'm still using the oldie 1.437 version where the same column gives me the best position in absolute value and not in a %. I can't read that %. Can anyone explain me this?

Sorry about the pics being in Portuguese, but I think you get the point.

View in version 6
![image](https://user-images.githubusercontent.com/12599665/222534327-d7529e65-b875-49b3-924c-7d335ea21259.png)

View in version 1.437
![image](https://user-images.githubusercontent.com/12599665/222535478-529c588c-bf51-4c9c-b01b-4f5e3ff2b9b1.png)

The only way I can get the same kind of value is when I have a player selected and I can get that value in the detail.
![image](https://user-images.githubusercontent.com/12599665/222534665-952c9fe8-60d1-43a3-9d19-2cf3f1a2db2e.png)

But it's not all. I have this player who is my best central defense player who usually gets when he's in a good shape a 8,5* classification in a game played. But, the best position value only gives him an 5,92 in HO6. This gets me confused a lot, as again I get myself comparing what was being done in the old 1.437 version where the calculation of this value is 7,74 for that same player which is much  more close to the game played value.

Best Regards to all,
Tiago

**Comment by wsbrenk** [comment]
The intention of the relative value was to normalize the different levels of the absolute rating information for the different areas attack, midfield, etc. to a common level. The percentage value should be a kind of relative value in relation to the maximum possible value.

**Reply by tiagompb** [reply]
I think I've gotten a little bit more confused, because the main question continues to be it's a % relative to what? Where can I see that max potential value for a specific position or that relative value? I don't see them in any column, just when I select a specific player and still I don't get the meaning of that value shown.
Are you suggesting that I have to go to all player's detail to get that understanding? If so, I think I'm going to keep using the oldie 1.437 HO version, which is quite absurd!
